From 0b645d407391f0791df1d77596f09f9a41795775 Mon Sep 17 00:00:00 2001
Message-Id: <0b645d407391f0791df1d77596f09f9a41795775.1510335500.git.Jim.Somerville@windriver.com>
From: Jim Somerville <Jim.Somerville@windriver.com>
Date: Thu, 9 Nov 2017 15:53:41 -0500
Subject: [PATCH 1/1] Fix compile warnings in scsi drivers

They get turned into errors and thus break the build.

Signed-off-by: Jim Somerville <Jim.Somerville@windriver.com>
---
 SPECS/kernel-rt.spec | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/SPECS/kernel-rt.spec b/SPECS/kernel-rt.spec
index 779f3ce..c8c7f22 100644
--- a/SPECS/kernel-rt.spec
+++ b/SPECS/kernel-rt.spec
@@ -419,6 +419,9 @@ Patch1025: US101216-IMA-support-in-Titanium-kernel.patch
 Patch1026: US103091-IMA-System-Configuration.patch
 Patch1027: timer-Reduce-timer-migration-overhead-if-disabled.patch
 Patch1028: timer-Minimize-nohz-off-overhead.patch
+# Fix compile warnings that break the build
+Patch1029: aic94xx-Skip-reading-user-settings-if-flash-is-not-f.patch
+Patch1030: dpt_i2o-fix-build-warning.patch
 
 BuildRoot: %{_tmppath}/kernel-%{KVERREL}-root
 
@@ -810,6 +813,8 @@ ApplyPatch US101216-IMA-support-in-Titanium-kernel.patch
 ApplyPatch US103091-IMA-System-Configuration.patch
 ApplyPatch timer-Reduce-timer-migration-overhead-if-disabled.patch
 ApplyPatch timer-Minimize-nohz-off-overhead.patch
+ApplyPatch aic94xx-Skip-reading-user-settings-if-flash-is-not-f.patch
+ApplyPatch dpt_i2o-fix-build-warning.patch
 
 # move off upstream version mechanism
 if [ -e localversion-rt ]; then
-- 
1.8.3.1

